client/src/pages/ToursToday.js

```diff
@@ -54,11 +54,9 @@ const ToursToday = () => {
                     <>
                       {ride.cargo && <Badge type={ride.cargo} active />}
                       {ride.priority !== "normal" &&
-                      ride.priority !== "concurrentRide" ? (
-                        <Badge type={ride.priority} active />
-                      ) : (
-                        ""
-                      )}
+                        ride.priority !== "concurrentRide" && (
```

Comment:
it's difficult to understand right away what this function is doing. 
Maybe it is helpful to extract this into a function or component?